html/webappapis/dynamic-markup-insertion/opening-the-input-stream/bailout-exception-vs-return-origin.sub.window.js

```diff
@@ -0,0 +1,82 @@
+document.domain = "{{host}}";
+
+async_test(t => {
+  const iframe = document.body.appendChild(document.createElement("iframe"));
+  t.add_cleanup(() => { iframe.remove(); });
+  iframe.onload = t.step_func(() => {
+    // Here, the entry settings object is still the iframe's. Delay it in such
+    // a way that makes the entry settings object the top-level page's, but
+    // without delaying too much that the parser becomes inactive. A microtask
```

Comment:
Maybe "...parser becomes inactive (which would trigger a separate bailout from the two that this test pits against each other)" or similar.

Also, extract this comment block into a function with a nice name, like `changeEntrySettingsObjectButWithoutWaitingTooLong(t.step_func_done(...))`, that you can reuse.